src/controllers/CheckAnswersController.js

```diff
@@ -14,54 +14,78 @@ class CheckAnswersController extends PageController {
    * @param {Object} req - The HTTP request object.
    * @param {Object} res - The HTTP response object.
    * @param {Function} next - The next middleware function.
-  */
-  post (req, res, next) {
-    this.sendEmails(req, res, next)
+   */
+  async post (req, res, next) {
+    const result = await this.sendEmails(req, res, next)
+    for (const err of (result.errors ?? [])) {
+      console.error(err)
```

Comment:
can be addressed in a later ticket, but as we have now merged the repos we have access to a logger. I'll add a tech debt ticket to update all console messages in the submit part of this repo to use the logger instead